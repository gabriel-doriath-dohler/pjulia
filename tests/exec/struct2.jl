struct test
    n::Int64
    b::Bool
	id::String
end

println(test(42, true, "yo").b)
x= test(42,true,"yo");print(x.n, "\n")
println(x.id)

