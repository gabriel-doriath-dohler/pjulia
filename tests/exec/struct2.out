true
42
yo
